### PR TITLE
Put correct version number in rustup version-requirement string

### PR DESCRIPTION
**Pull request by pshaughn**
Just fixing a little typo that confused me when I saw it come up.

### CODE REVIEW

File: python/servo/command_base.py [hunk 1]
```diff
@@ -1037,7 +1037,7 @@ def ensure_rustup_version(self):
             raise
         version = tuple(map(int, re.match(b"rustup (\d+)\.(\d+)\.(\d+)", version_line).groups()))
         if version < (1, 21, 0):
-            print("rustup is at version %s.%s.%s, Servo requires 1.11.0 or more recent." % version)
+            print("rustup is at version %s.%s.%s, Servo requires 1.21.0 or more recent." % version)
```
Comment (jdm) [inline python/servo/command_base.py]: Let's make this a variable that's used in the check as well to avoid this happening again in the future.

**Review by jdm — CHANGES_REQUESTED** [review]

**Comment by SimonSapin** [comment]
Ouch, I specifically thought of not making this mistake and then went on to do it. Sorry!

Also, `return 1` in this position is not effective anymore. (With this section of code having moved into another method.) Should this check failing call `sys.exit` or raise an exception instead?

**Comment by jdm** [comment]
```
./python/servo/command_base.py:1039: E703 statement ends with a semicolon
```

**Comment by pshaughn** [comment]
Sorry about that!

**Pull request merged**